UIKit_DZ/UIKit_DZ/BagSection/BagItem/BagItemView.swift

```diff
@@ -0,0 +1,203 @@
+        costLabel.text = "\(text)" + Constants.currencySigh
+    }
+
+    func setAmount(_ amount: Int) {
+        amountStepper.setAmount(amount)
+    }
+
+    func setSize(_ size: Int) {
+        sizeSelectorView.setSelectedSize(size)
+    }
+
+    func toggleCartIcon(to value: Bool) {
+        goodItemView.setCartButtonHighlition(to: value)
+    }
+
+    // MARK: - Private Methods
+
+    private func configureUI() {
+        addSubviews(nameLabel, amountLabel, sizeLabel, costTitleLabel, costLabel)
+        addSubviews(goodItemView, amountStepper, sizeSelectorView)
+    }
+
+    private func configureLayout() {
+        UIView.doNotTranslateAoturesizingMaskIntoConstrains(for: subviews)
+        [
+            goodItemView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            goodItemView.topAnchor.constraint(equalTo: topAnchor, constant: -18),
+            goodItemView.bottomAnchor.constraint(equalTo: bottomAnchor),
+            goodItemView.widthAnchor.constraint(equalTo: goodItemView.heightAnchor),
+
+            nameLabel.leadingAnchor.constraint(equalTo: goodItemView.trailingAnchor, constant: 16),
+            nameLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10),
+            nameLabel.topAnchor.constraint(equalTo: topAnchor),
+
+            amountLabel.leadingAnchor.constraint(equalTo: goodItemView.trailingAnchor, constant: 16),
+            amountLabel.topAnchor.constraint(equalTo: nameLabel.bottomAnchor, constant: 12),
+
+            amountStepper.leadingAnchor.constraint(equalTo: amountLabel.trailingAnchor, constant: 10),
+            amountStepper.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10),
+            amountStepper.topAnchor.constraint(equalTo: nameLabel.bottomAnchor, constant: 12),
+            amountStepper.heightAnchor.constraint(equalToConstant: 15),
+
+            sizeLabel.leadingAnchor.constraint(equalTo: goodItemView.trailingAnchor, constant: 16),
+            sizeLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10),
+            sizeLabel.topAnchor.constraint(equalTo: amountLabel.bottomAnchor, constant: 12),
+
+            sizeSelectorView.leadingAnchor.constraint(equalTo: goodItemView.trailingAnchor, constant: 22),
+            sizeSelectorView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10),
+            sizeSelectorView.topAnchor.constraint(equalTo: sizeLabel.bottomAnchor, constant: 10),
+
+            costTitleLabel.leadingAnchor.constraint(equalTo: goodItemView.trailingAnchor, constant: 16),
+            costTitleLabel.topAnchor.constraint(equalTo: sizeSelectorView.bottomAnchor, constant: 7),
+
+            costLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10),
+            costLabel.topAnchor.constraint(equalTo: sizeSelectorView.bottomAnchor, constant: 9),
+        ].activate()
+    }
+}
+
+///  Получение увадомлений о нажатии кнопки корзины
+extension BagItemView: GoodItemViewDelegate {
```

Comment:
```suggestion

///  Получение `увадомлений` о нажатии кнопки корзины
extension BagItemView: GoodItemViewDelegate {
```
грамматическая ошибка